tests/ethereum/test_regressions.py

```diff
@@ -273,7 +273,7 @@ def test_addmod(self):
         caller = 0x42424242424242424242
         value = 0
         bytecode = ""
-        vm = evm.EVM(constraints, address, data, caller, value, bytecode)
+        vm = evm.EVM(constraints, address, data, caller, value, bytecode, gas=23000)
```

Comment:
2300?

Same question applies to line 333 below.

Reply:
I think just wanted the execution to have enough gas.